### ISSUE TITLE
When launching Browser, Addon sometimes does not function properly.

### ISSUE THREAD

**Issue by Shitennouji** [JustOff/advanced-night-mode#6]
In Pale Moon 28.4.0 (Win 64bit), "Advanced Night Mode 1.0.9" does not function properly every time the browser is activated.
In that case, the add-on icon is gone. It does not exist anywhere.
Therefore, in order to function properly, you need to remove and reinstall the add-on using the add-on manager.
However, there are times when the same symptoms are reproduced at the next startup and it is necessary to repeat the same method.

It appears to be in conflict with other add-ons, but is it possible to fix it?

By the way, the add-ons I have implemented are as follows.
Dark Moon 2.3.0
Add-ons Manager Context Menu 0.4.2.1-signed.1-signed
Autoplay Toggle (Non-Restartless) 1.0.1
BarTab Tycho 4.0
Block Content 0.3
Calendate 4.0.1
Cookie Masters 3.2.0
Cookies Exterminator 2.9.4
Decentraleyes 1.4.2
Greedy Cache 1.2.0
HideScrollbars 1.1
Lull The Tabs 1.4.3
ScriptBlock 1.0.0
Self-Destructing Cookies for Pale Moon 1.0.4.13
Tree Style Tab 0.0.5
uBlock Origin 1.16.4.8
uBlock Origin Updater 1.6.6
uMatrix 1.0.0

**Comment by JustOff** [comment]
Yes, it is quite likely this is a conflict of extensions, but unfortunately I can hardly help you until you figure out which one and provide steps to reproduce the issue.

**Comment by Shitennouji** [comment]
Thank you for your prompt reply.
I understood the content of the response.
I will explore in safe mode etc.
But private time is limited to me. A considerable number of days is required for investigation.
I will contact you again if I can identify it.

**Comment by Shitennouji** [comment]
The issue seems to have been resolved.
I repeatedly restarted Browser, but the issue is no longer reproduced and it is functioning normally.
In the meantime, it seems that there was a cause in Addons where Update was executed.

I reinstalled the old add-on before updating and tested it,
It is estimated that "Calendate 4.0.1" is the cause.
From that "Calendate 4.0.2" onwards, the problem will not be reproduced.
Its latest version is "Calendate 4.1", but of course, it did not have any problem.
I will report on the above.
